app/controllers/application_controller/ci_processing.rb

```diff
@@ -2511,6 +2478,18 @@ def refreshhosts
     host_button_operation('refresh_ems', _('Refresh'))
   end
 
+  def refreshphysicalservers
```

Comment:
Same comment as above.
It's best to handle all `Physical Server` actions in `physical_server_controller`